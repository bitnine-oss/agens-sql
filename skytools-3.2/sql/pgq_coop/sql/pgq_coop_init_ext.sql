
create extension pgq;

\set ECHO none
\i structure/install.sql
\set ECHO all

create extension pgq_coop from 'unpackaged';
drop extension pgq_coop;

create extension pgq_coop;

