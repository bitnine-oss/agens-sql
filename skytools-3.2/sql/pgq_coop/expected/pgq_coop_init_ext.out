create extension pgq;
\set ECHO none
create extension pgq_coop from 'unpackaged';
drop extension pgq_coop;
create extension pgq_coop;
